api/resolvers/search.js

```diff
@@ -411,6 +387,24 @@ export default {
               fields: ['title^10', 'text'],
               boost: 1000
             }
+          },
+          // match on exact fields higher
+          {
+            multi_match: {
+              query,
+              type: 'best_fields',
+              fields: ['title.exact^10', 'text.exact'],
+              boost: 100
+            }
+          },
+          // exact phrase matches higher
+          {
+            multi_match: {
+              query,
+              fields: ['title.exact^10', 'text.exact'],
+              type: 'phrase',
+              boost: 10000
+            }
```

Comment:
I also thought it'd be nice to use these exact fields in other queries to boost relevance.